src/lib/Preview.js

```diff
@@ -1360,6 +1398,9 @@ class Preview extends EventEmitter {
 
             // If there wasn't an error and event logging is not disabled, use Events API to log a preview
             if (!this.options.disableEventLog) {
+                if (this.options.enableAdvancedContentInsights) {
```

Comment:
Does this need to be tied to `disableEventLog`?

Reply:
As per our requirements yes, because we won’t report the ACI if the access-stats are disabled, because we need to correlate the data.